src/agentlessd/agentlessd.c

```diff
@@ -8,6 +8,10 @@
  * Foundation
  */
 
+#ifdef WAZUH_UNIT_TESTING
+#define static
+#endif
+
```

Comment:
Desing severe: replacing a C keyword with nothing is not recommended. Check `STATIC` usage on other already implemented UT

Reply:
maybe wouldn't be the optimal solution, but is implemented in every source code that needs a UT.

Reply:
Only 5 UT use this approach. The other ones use the next one
```c
#ifdef WAZUH_UNIT_TESTING
// Remove STATIC qualifier from tests
#define STATIC
#else
#define STATIC static
#endif
```